plugins/jobtype/src/azkaban/jobtype/HadoopPigJob.java

```diff
@@ -112,6 +113,11 @@ public void run() throws Exception {
       Props props = new Props();
       props.putAll(getJobProps());
       props.putAll(getSysProps());
+      String additionalNamenodes =
```

Comment:
Nearly all of the code is duplicated between Hadoop(Pig,Hive,Java)Job. They really should inherit from a common subclass. I can do that as part of this PR if you'd like.